setup.json

```diff
@@ -4,7 +4,10 @@
             "module": "iquip.apps.explorer",
             "cls": "ExplorerApp",
             "show": true,
-            "pos": "left"
+            "pos": "left",
+            "args": {
+                "masterPath": "/home/jaehun/quiqcl/artiq/examples/"
```

Comment:
How about setting this as the default ARTIQ master path?

Reply:
Umm.. Unfortunately there is no default path because `artiq_dashboard` can obtain the master path using the internal socket communication.
Then, how about setting it as the path in mini PC?

Reply:
Oh, I was referring to that! Let's set it to the mini PC path.